### DISCUSSION TITLE
BthPS3 prevents Nintendo Switch PRO controller to reconnect

### DISCUSSION THREAD

**Discussion by tij77** [nefarius/BthPS3#37] [Support]
Tried BthPS3 as it seems the only viable solution to get all DS3 features to PCSX2 on Windows 10 (together with DsHIDmini). And it works like a charm.

The problem is BthPS3 prevents other controller to reconnect. The controller in question is GameSir T4 PRO that connects to PC via bluetooth as Nintendo Switch PRO controller.

Before BthPS3 - that controller worked fine. With BthPS3:
1. Adding controller to Windows 10 (via "+Add Bluetooth & other devices") works fine and it is added as "Pro Controller". After its added, it stays "Connected" and functions as normal
2. Disconnecting controller changes its state to "Paired". Reconnecting controller back is not possible ("Paired" state changes to "Connected" for few seconds then changes back to "Paired")
3. Turning off "Enable PSM patching" in BthPS3 Driver Configuration Utility solves the issue, but then I looses ability to use DS3

Anything i can do to fix this?

**Comment by nefarius** [comment]
The fix is literally in your own post, it's your Nr. 3, and no, all you lose is DS3 support **during wireless connection attempt**, which is a small price to pay. As mentioned in the FAQs, this is an unavoidable inconvenience you have to live with.

Cheers

**Reply by tij77** [reply]
Thanks for the quick answer. Should have read FAQ first ... my fault.

Its indeed a small price to pay ... and with BthPS3Util.exe I can automate it ... for total couch experience :)

If only something can be exposed for DsHIDmini ... so can automate it switching between SXS and SDF modes as discussed in this thread https://github.com/ViGEm/DsHidMini/discussions/160 :)

**Reply by nefarius** [reply]
It will come the day I find the time to work on it, as simple as that 😉

**Reply by tij77** [reply]
Thanks for your great work
